src/main/java/at/hannibal2/skyhanni/features/commands/WikiManager.kt

```diff
@@ -61,17 +59,22 @@ class WikiManager {
         if (NEUItems.neuHasFocus()) return //because good heavens if this worked on neuitems...
         val stack = gui.slotUnderMouse?.stack ?: return
 
-        if (!config.fandomWikiKeybind.isKeyHeld()) return
-        wikiTheItem(stack)
+        if (!config.wikiKeybind.isKeyHeld()) return
+        wikiTheItem(stack, config.menuOpenWiki)
     }
 
-    private fun wikiTheItem(item: ItemStack) {
+    private fun wikiTheItem(item: ItemStack, autoOpen: Boolean) {
+        val urlPrefix = if (config.useFandom) "https://hypixel-skyblock.fandom.com/wiki/" else "https://wiki.hypixel.net/"
+        val urlSearchPrefix = if (config.useFandom) "${urlPrefix}Special:Search?query=" else "${urlPrefix}index.php?search="
+
         val itemDisplayName = (item.nameWithEnchantment ?: return).replace("§a✔ ", "").replace("§c✖ ", "")
         val internalName = item.getInternalName().asString()
-        LorenzUtils.chat("Searching the Fandom Wiki for §a$itemDisplayName")
-        val wikiUrlSearch = if (internalName != "NONE") "$urlSearchPrefix$internalName&scope=internal"
-        else "$urlSearchPrefix${itemDisplayName.removeColor()}&scope=internal"
-        OSUtils.openBrowser(wikiUrlSearch.replace(' ', '+'))
+        val wikiUrlSearch = if (internalName != "NONE") "$urlSearchPrefix$internalName"
```

Comment:
Please url encode the internal name

Reply:
Is the url encode the `.replace(' ', '+')`?

Reply:
See: https://docs.oracle.com/javase/8/docs/api/java/net/URLEncoder.html (or https://url.spec.whatwg.org/#percent-encoded-bytes)